### PR TITLE
Inform users of bad builds

### PR DESCRIPTION
**Pull request by jacob-delgado**
Inform users that a previous CVE was not included in the latest 1.13.6 and 1.14.2 builds.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [X] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Review by craigbox — COMMENTED** [review]
I think this should be news, not a blog post?

File: content/en/docs/releases/supported-releases/index.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ Please keep up-to-date and use a supported version.
 
 | Minor Releases   | Patched versions with no known CVEs           |
 |------------------|-----------------------------------------------|
-| 1.14.x           | 1.14.1+                                       |
-| 1.13.x           | 1.13.5+                                       |
+| 1.14.x           | 1.14.1 (not 1.14.2)                           |
+| 1.13.x           | 1.13.5 (not 1.13.5)                           |
```
Comment (lei-tang) [inline content/en/docs/releases/supported-releases/index.md:81]: Should "not 1.13.5" be "not 1.13.6" instead?

File: content/en/news/releases/1.13.x/announcing-1.13.6/index.md [hunk 2]
```diff
@@ -7,6 +7,11 @@ publishdate: 2022-07-25
 release: 1.13.6
 ---
 
+{{< warning >}}
+Istio 1.13.6 does not contain a fix for [CVE-2022-31045](news/security/istio-security-2022-005/#cve-2022-31045). We recommend users do not install Istio 1.13.6
```
Comment (ericvn) [inline content/en/news/releases/1.13.x/announcing-1.13.6/index.md:11]: c/do/to or remove word altogether.

File: content/en/news/releases/1.14.x/announcing-1.14.2/index.md [hunk 3]
```diff
@@ -7,6 +7,11 @@ publishdate: 2022-07-25
 release: 1.14.2
 ---
 
+{{< warning >}}
+Istio 1.14.2 does not contain a fix for [CVE-2022-31045](news/security/istio-security-2022-005/#cve-2022-31045). We recommend users do not install Istio 1.14.2
```
Comment (ericvn) [inline content/en/news/releases/1.14.x/announcing-1.14.2/index.md:11]: c/do/to or remove word altogether.

**Comment by ericvn** [comment]
> I think this should be news, not a blog post?

I agree. people will see the new releases on the news back and not see this at all.

File: content/en/docs/releases/supported-releases/index.md (hunk 1 above)
Comment (jacob-delgado) [reply content/en/docs/releases/supported-releases/index.md:81]: Fixed

File: content/en/news/security/istio-security-2022-006/index.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
+---
+title: ISTIO-SECURITY-2022-006
+subtitle: Security Bulletin
+description: Ill-formed headers sent to Envoy in certain configurations can lead to unexpected memory access resulting in undefined behavior or crashing.
+cves: [CVE-2022-31045]
+cvss: "5.9"
+vector: "CVSS:3.1/AV:N/AC:H/PR:N/UI:N/S:U/C:N/I:N/A:H"
+releases: ["1.13.6", "1.14.1"]
```
Comment (ericvn) [inline content/en/news/security/istio-security-2022-006/index.md:8]: 1.14.2

**Review by ericvn — APPROVED** [review]

**Review by lei-tang — APPROVED** [review]

**Pull request merged**